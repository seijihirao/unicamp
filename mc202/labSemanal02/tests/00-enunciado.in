15
7
4 2 2 4 3 1 3
