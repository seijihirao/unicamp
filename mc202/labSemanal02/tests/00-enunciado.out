20 22 22
